src/Heap.js

```diff
@@ -5,16 +5,27 @@ export default class Heap {
     get length() {
         return this.heap.length;
     }
+    
+    checkNode(node) {
```

Comment:
so – what does `checkNode` mean? sort of similarly, many of the methods in here don't match traditional names

e.g., `checkNode` should probably be `has` or `includes` or `contains`, and `extractRoot`, while *fine*, could also be a `pop`

Reply:
Well checkNode isn't actually used, so i think I'll delete it. extractRoot as pop makes it a lot easier to see what is going on though so I'll do that. 🐇